.github/workflows/tests.yml

```diff
@@ -12,17 +12,22 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - uses: actions/setup-python@v5
+    # - name: Setup tmate session
+    #   uses: mxschmitt/action-tmate@v3
       with:
         python-version: '3.11'
-    - name: Install gdb gdbserver openmpi
+    - name: Install system dependencies
       run: |
         sudo apt update
-        sudo apt install -yq gdb gdbserver libopenmpi-dev
+        sudo apt install -y gdb libopenmpi-dev libssl-dev
     - name: Build example mpi program
       run: |
         make -C examples/
     - name: Install mdb and additional python packages
       run: |
+        mkdir -p ~/.mdb
+        openssl req -x509 -newkey rsa:4096 -sha256 -days 365 -keyout ~/.mdb/key.rsa -out ~/.mdb/cert.pem -nodes -subj "/C=XX/ST=mdb/L=mdb/O=mdb/OU=mdb/CN=localhost"
```

Comment:
Shouldn't the exchange server do this when it starts up?

Reply:
it should... this is from my debugging. I will leave this unresolved until I remove this garbage

Reply:
I have removed it :+1: